.github/workflows/scanner-offline-bundle-update.yaml

```diff
@@ -40,31 +51,74 @@ jobs:
       with:
         gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
 
-    - name: Authenticate with Google Cloud
+    - name: Generate offline bundle
+      run: |
+        .github/workflows/scripts/scanner-update-offline-bundle.sh \
+            bundle \
+            "$VULNERABILITY_VERSION" \
+            "$SUPPORTED_RELEASES"
+
+    - uses: ./.github/actions/upload-artifact-with-retry
+      with:
+        name: offline-bundle-${{ matrix.versions.vulnerability_version }}
+        path: bundle
+        if-no-files-found: error
+
+  upload-bundle:
+    runs-on: ubuntu-latest
+    needs:
+    - generate-bundle
+    steps:
+    - uses: actions/download-artifact@v4
+      with:
+        path: bundles
+        pattern: offline-bundle-*
+        merge-multiple: true
+
+    - name: Authenticate with Google Cloud (release)
+      if: github.event_name == 'schedule'
       uses: google-github-actions/auth@v2
       with:
         credentials_json: ${{ secrets.GOOGLE_SA_STACKROX_HUB_VULN_DUMP_UPLOADER }}
 
+    - name: Authenticate with Google Cloud (test)
```

Comment:
nit: use the same 'word' in the parenthesis between this step and the corresponding one later one

Current:
```
Authenticate with Google Cloud (test)
Upload bundles (pull request)
```
Perhaps:
```
Authenticate with Google Cloud (pull request)
Upload bundles (pull request)
```

Reply:
Nice suggestion, :+1: